content/docs/concepts/network/index.md

```diff
@@ -0,0 +1,437 @@
+are not limited to external service configuration. A service entry can have one
+of two types: mesh-internal or mesh-external:
+
+-  **Mesh-internal service entries** explicitly add internal services to the
+   Istio mesh. You can use mesh-internal service entries to add services as
+   your service mesh expands to include unmanaged infrastructure. The unmanaged
+   infrastructure can include components such as VMs added to a
+   Kubernetes-based service mesh.
+
+-  **Mesh-external service entries** explicitly add external services to the
+   mesh. Mutual TLS authentication is disabled for mesh-external service
+   entries. Istio performs policy enforcement on the client-side, instead of on
+   the usual server-side used by internal service requests.
+
+Mesh-external service entries provide the following configuration options:
+
+-  You can configure multiple external dependencies with a single service
+   entry.
+-  You can configure the resolution mode for the external dependencies to
+   `NONE`, `STATIC`, or `DNS`.
+-  You can access secure external services over plain text ports to directly
+   access external dependencies from your application.
+
+The following example configuration shows a mesh-external service entry for the
+`ext-resource` external dependency:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
+metadata:
+  name: svc-entry
+spec:
+  hosts:
+  - ext-resource.com
+  ports:
+  - number: 443
+    name: https
+    protocol: HTTPS
+  location: MESH_EXTERNAL
+{{< /text >}}
+
+Because a service entry configuration simply adds the external resource to the
+internal service registry, you must use either a virtual service or destination
+rules to complete the configuration. You configure a service entry similarly to
+how you configure a service in the mesh. The following destination rule
+configures the traffic route to use mutual TLS to connect to the `ext-resource`
+external service that the service entry added to the mesh:
+
+{{< text yaml >}}
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: ext-res-dr
+spec:
+  host: ext-resource.com
+  trafficPolicy:
+    tls:
+      mode: MUTUAL
+      clientCertificate: /etc/certs/myclientcert.pem
+      privateKey: /etc/certs/client_private_key.pem
+      caCertificates: /etc/certs/rootcacerts.pem
+{{< /text >}}
+
+Together, the `svc-entry` service entry and the `ext-res-dr` destination rule
+configure a route for the  HTTPS traffic to and from the `ext-resource`
+external dependency through the `svc-entry` service entry on port 80 and using
+mutual TLS. The following diagram shows the configured traffic routes:
+
+{{< image width="50%"
+    link="./net-config-8.svg"
+    caption="Configurable traffic routes using service entries and destination rules"
+    >}}
+
+Service entries do not require virtual services to address services in the
+mesh. Use destination rules to configure routes to address services directly.
```

Comment:
Not sure what this is trying to say, but I think it's wrong.

Reply:
It was trying to say what you stated in the previous comment only awkwardly. I've removed it.